runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -1368,9 +1369,10 @@ J9::ARM64::TreeEvaluator::VMnewEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    // 5. Allocate object/array on heap
    genHeapAlloc(node, cg, isVariableLength, allocateSize, elementSize, resultReg, lengthReg, tempReg1, tempReg2, tempReg3, conditions, callLabel);
 
-   // 6. Setup HeapAllocSnippet for slowpath
-   TR::Snippet *snippet = new (cg->trHeapMemory()) TR::ARM64HeapAllocSnippet(cg, node, callLabel, node->getSymbolReference(), doneLabel);
```

Comment:
Is `ARM64HeapAllocSnippet` used anywhere else?
If not, please remove its code from J9ARM64Snippet.hpp/cpp.  You can do it in a separate PR.

Reply:
The heap alloc snippet is not used anywhere. I will remove it in a separate PR.